api/notd/manager.py

```diff
@@ -226,6 +231,7 @@ async def process_block(self, blockNumber: int) -> None:
         await asyncio.gather(*[self._create_token_transfer(retrievedTokenTransfer=retrievedTokenTransfer) for retrievedTokenTransfer in retrievedTokenTransfers])
         for retrievedTokenTransfer in retrievedTokenTransfers:
             await self.workQueue.send_message(message=UpdateTokenMetadataMessageContent(registryAddress=retrievedTokenTransfer.registryAddress, tokenId=retrievedTokenTransfer.tokenId).to_message())
+            await self.workQueue.send_message(message=UpdateCollectionMessageContent(address=retrievedTokenTransfer.registryAddress).to_message())
```

Comment:
we shouldnt do this - it will cause lots of duplcates. you should gather a set of the registryAddresses and send the message in a separate for loop